.github/workflows/build.yml

```diff
@@ -470,8 +470,6 @@ jobs:
           cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT-sources.jar artifacts
           cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT-javadoc.jar artifacts
 
-          cp liquibase-dist/target/liquibase-*.deb artifacts
```

Comment:
@sayaliM0412 this is the line that fails here https://github.com/liquibase/liquibase/actions/runs/5884307781/job/15959135849#step:9:366